dist/manifest.json

```diff
@@ -12,7 +12,8 @@
     "scripting",
     "notifications",
     "declarativeNetRequest",
-    "declarativeNetRequestFeedback"
+    "declarativeNetRequestFeedback",
+    "alarms"
```

Comment:
I want to know the documentation related to this permission. Please provide me the doccs